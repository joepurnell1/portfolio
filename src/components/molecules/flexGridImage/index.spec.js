import React from 'react';
import renderer from 'react-test-renderer';
import FlexGridImage from './index';

describe('FlexGridImage component', () => {
  it('renders default as expected', () => {
    const render = renderer.create(
      <FlexGridImage
        imageSource="https://img.discogs.com/6AAosovSxIH2OIGpql6fZa9I2QE=/fit-in/600x600/filters:strip_icc():format(jpeg):mode_rgb():quality(90)/discogs-images/R-5617558-1398104965-3485.png.jpg"
        imageAlt="Still looking for that blue jean, baby queen"
      />,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });

  it('renders custom size as expected', () => {
    const render = renderer.create(
      <FlexGridImage
        imageSource="https://img.discogs.com/6AAosovSxIH2OIGpql6fZa9I2QE=/fit-in/600x600/filters:strip_icc():format(jpeg):mode_rgb():quality(90)/discogs-images/R-5617558-1398104965-3485.png.jpg"
        imageAlt="Prettiest girl I ever seen"
        imageSize="225px"
        containerSize="250px"
      />,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});
