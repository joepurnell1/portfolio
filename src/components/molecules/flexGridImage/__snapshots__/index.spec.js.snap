// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlexGridImage component renders custom size as expected 1`] = `
<div
  className="sc-bdVaJa cokHZi"
  size="250px"
>
  <img
    alt="Prettiest girl I ever seen"
    className="sc-bwzfXH cuLAQ"
    height="225px"
    src="https://img.discogs.com/6AAosovSxIH2OIGpql6fZa9I2QE=/fit-in/600x600/filters:strip_icc():format(jpeg):mode_rgb():quality(90)/discogs-images/R-5617558-1398104965-3485.png.jpg"
    width="225px"
  />
</div>
`;

exports[`FlexGridImage component renders default as expected 1`] = `
<div
  className="sc-bdVaJa gJHIfT"
  size="150px"
>
  <img
    alt="Still looking for that blue jean, baby queen"
    className="sc-bwzfXH bcBBVT"
    height="125px"
    src="https://img.discogs.com/6AAosovSxIH2OIGpql6fZa9I2QE=/fit-in/600x600/filters:strip_icc():format(jpeg):mode_rgb():quality(90)/discogs-images/R-5617558-1398104965-3485.png.jpg"
    width="125px"
  />
</div>
`;
